apps/backend/src/sales/dto/payment.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { IsOptional } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString, Length, IsNumber } from 'class-validator';
+
+export class PaymentsDTO {
+  @ApiProperty({ description: 'Amount paid', example: 100.5 })
+  @ApiProperty()
+  @IsNumber()
+  @IsNotEmpty()
+  amount!: number;
+
+  @ApiProperty({
+    description: 'Method Of Payment',
+    example: '10'
```

Comment:
This should be an ENUM. 

I think there is also a way to add the enum in the swagger?